src/components/Preview.jsx

```diff
@@ -13,7 +13,7 @@ class Preview extends React.Component {
     bindAll(this, '_handlePopOutClick');
   }
 
-  _generateDocument() {
+  _generateDocument(isPopout = false) {
```

Comment:
Not sure I follow - the popout action calls this with true, the in window preview calls it with false. The other 3 parameters should always be true, or at least that seems to be the behavior before this change that I'm making.